SchoolManagement/src/main/java/com/school/management/dao/DivisionDaoImpl.java

```diff
@@ -72,10 +78,10 @@ public Division getDivision(Long division_id) {
 	}
 
 	@Override
-	public List<Division> getDivisionList() {
+	public List<Division> getDivisionList(Long standard_id) {
 		List<Division> divisionList = new ArrayList<>();
 		try {
-			divisionList = divisionRepository.findAll();
+			divisionRepository.findByStandardStandard_id(standard_id).forEach(divisionList::add);
 		} catch(Exception e) {
 			String error = String.format("Error occured while fetching division list.");
 			logger.error(error);

```

Comment:
sould we log exception also like `logger.error(error,e);`

Reply:
Yes. We can use exception object as well.